util/EIP-1024/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Generated by Cargo
```

Comment:
It is better that you add this crate to the workspace instead to avoid having multiple `Cargo.lock` generated until this crate is used in the dependency tree.

Reply:
ok